### ISSUE TITLE
iOs: Music Controls always start with the pause icon, regardless of 'isPlaying' setting.

### ISSUE THREAD

**Issue by jrarcher** [ghenry22/cordova-music-controls-plugin#31]
```
this.musicCtrl.create({
      track: track_title,
      artist: this.book.author,
      cover: this.book.thumbURL,
      isPlaying: false,
      dismissable: true
     }) 
```

Irrespective of whether or not I call:
 `this.musicCtrl.updateElapsed({ isPlaying: false, elapsed: this.trackPosition });` or
`this.musicCtrl.updateIsPlaying(false);` 

The music control always shows the lock screen controls with the paused icon at the beginning. It only seems to honor a touch event to change the icon.

Any ideas?

**Comment by ghenry22** [comment]
Just want to check but in all your examples you are setting isPlaying to false?

**Comment by Holubinka** [comment]
@ghenry22 I setting isPlaying to true and i have the same problem

**Comment by jrarcher** [comment]
@ghenry22 That is correct. We toggle it in places, of course, but as an experiment I set 'isPlaying' to false everywhere and under no scenario will the 'play' icon appear when we first go to the lock screen.

**Comment by ghenry22** [comment]
After looking at this I feel like the isPlaying values should really be removed as they are a bit misleading and don't work as you would expect.

No matter what you set in this plugin the actual audio status (and hence what is shown in the remote controls) will be set by the playing media and the media session itself and will override this.  So if you remove the isPlaying variable from your code and just set the audio itself to either playing or paused then the remote controls will reflect that state correctly.

Does that make sense?

**Comment by jrarcher** [comment]
@ghenry22 That makes sense. I'll try that approach this week and report back what I find. Thanks for taking the time to look at this.

**Comment by Holubinka** [comment]
@ghenry22  I removed isPlaying, updateIsPlaying and updateElapsed anywhere but this not helped me. But I use with this plugin cordova-plugin-media.

**Comment by ghenry22** [comment]
@Holubinka the plugin works perfectly with the cordova media plugin.

Create your media
On play create your music controls

Your music controls will reflect the state of your media automatically (ie if your media is playing then the music controls will show the pause icon).

I use the plugin extensively in some of my own apps along with the cordova media plugin and it works very well.

Note that if you have the background mode plugin installed it will often interfere with the playback status as it plays a "fake" audio in the background.

I have a forked version of the background mode plugin without the faked audio behaviour that fixes this, I'll provide a link to it later when I can check that my fork is up to date.

**Comment by Holubinka** [comment]
@ghenry22 OK. Thank you I'll wait your link for forked version. I'll try create music controls when media is playing.

**Comment by julianCast** [comment]
> I have a forked version of the background mode plugin without the faked audio behaviour that fixes this, I'll provide a link to it later when I can check that my fork is up to date.

Do you have the link @ghenry22 ?

**Comment by Holubinka** [comment]
> @Holubinka the plugin works perfectly with the cordova media plugin.
> 
> Create your media
> On play create your music controls
> 
> Your music controls will reflect the state of your media automatically (ie if your media is playing then the music controls will show the pause icon).
> 
> I use the plugin extensively in some of my own apps along with the cordova media plugin and it works very well.
> 
> Note that if you have the background mode plugin installed it will often interfere with the playback status as it plays a "fake" audio in the background.
> 
> I have a forked version of the background mode plugin without the faked audio behaviour that fixes this, I'll provide a link to it later when I can check that my fork is up to date.

@ghenry22 Please provide link to your background plugin

**Comment by ghenry22** [comment]
@Holubinka https://github.com/ghenry22/cordova-plugin-background-mode
This background mode plugin fork removes the fake audio playback on iOS

**Comment by MuhammadSaadBukhtiar97** [comment]
Hello everyone,

I have tried all above mentioned methods, but unable to change state of icon from play to pause or pause to play of media in notificattion bar of ios.
